src/server.cpp

```diff
@@ -499,6 +502,7 @@ void Server::step(float dtime)
 			throw ServerError(async_err);
 		}
 		else {
+			m_env->kickAllPlayers("A server LUA stack crash has occurred. Disconnecting all players. Please reconnect soon...");
```

Comment:
maybe this should be to be a bit more noob freindly, most people wont know what a lua stack is:
`m_env->kickAllPlayers("The server has crashed. Disconnecting all players. Please reconnect soon...");`

:)